reckless_cmd/src/reckless/mod.rs

```diff
@@ -17,7 +17,9 @@ mod config;
 
 pub struct RecklessManager {
     config: config::RecklessConf,
-    repos: Vec<Box<dyn Repository>>,
+    /// List of repositories
+    repos: Vec<Box<Github>>,
```

Comment:
No this is wrong! The previous one was correct!

With your solution, you are breaking all the architecture because we can not work with a repository that it is different from the Github one